### PR TITLE
Made amendment free text show on projection

### LINKED ISSUE #2218: Project free text amendment does not work
* Set option "empty text field" in motion settings (How to create new amendments)
* create a new amendment (with some free text)
* project this amendment
=> projector shows "No changes at the text"
expected: show entered amendment text (without any diff view!)

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #2218

### CODE REVIEW

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
If you disable the motion settings "Show motion text on projector" the projected free text  amendment still shows "No changes on the text":

amendment-free-projection

**Comment by luisa-beerboom** [comment]
> If you disable the motion settings "Show motion text on projector" the projected free text amendment still shows "No changes on the text":
> 
> amendment-free-projection

That's an autoupdate problem

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-detail-diff/motion-detail-diff.component.ts [hunk 1]
```diff
@@ -209,6 +209,7 @@ export class MotionDetailDiffComponent extends BaseMeetingComponent implements A
                 firstLine: this.motion.firstLine
             });
         }
+        console.log(`BASETEXT`, baseText, this.getAllTextChangingObjects());
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-detail-diff/motion-detail-diff.component.ts:212]: Debug output should be removed

**Review by bastianjoel — CHANGES_REQUESTED** [review]

**Review by emanuelschuetze — DISMISSED** [review]
The projector shows still amendment text if settings "Show motion text on projector"  is disabled.

**Comment by luisa-beerboom** [comment]
> The projector shows still amendment text if settings "Show motion text on projector" is disabled.

So what do you want the projector to show for text amendments if the setting is disabled?

**Comment by luisa-beerboom** [comment]
Update: It should not show any text at all.
No "The assembly may decide" and no "No changes at the text", just title and submitters

**Review by bastianjoel — APPROVED** [review]

**Comment by luisa-beerboom** [comment]
Update after talking to @emanuelschuetze 
Disabling motion texts on projector _should_ disable any and all text _including_ diff lines for paragraph based amendments. The entire block down from and including "The assembly may decide" should be gone regardless of motion type, as soon as the setting is disabled!

TODO:
- [ ] Change the component to never show that block when the setting is disabled.
- [ ] Make an autoupdate issue/PR that removes the amendment paragraph information as well if the setting is disabled (since it is unnecessary at that point)

**Review by bastianjoel — APPROVED** [review]

**Comment by luisa-beerboom** [comment]
>     * [x]  Make an autoupdate issue/PR that removes the amendment paragraph information as well if the setting is disabled (since it is unnecessary at that point)

=> OpenSlides/openslides-autoupdate-service#687

**Comment by luisa-beerboom** [comment]
Merged upon request

**Pull request merged**